hecate.yaml

```diff
@@ -165,15 +165,24 @@ aws:
 ########################################
 # Argus - Webhooks Config
 ########################################
-argus:
-  # bucket is the partition name for storing webhooks.
+
+# migration describes how items should be pushed to Argus. 
+# (Optional) All the values below will be used as default 
+# if none is specified.
+migration:
+  # owner will be the owner of the items inserted into Argus from SNS.
+  # On the reading side of Argus then components can use this value to fetch
+  # migrated items.
```

Comment:
we should explicitly say that these two values are optional and state their defaults.